.github/workflows/task-build-artifacts-with-config.yml

```diff
@@ -0,0 +1,51 @@
+name: Build Artifacts with Platform Configuration
+
+# This workflow gets configuration for a platform/architecture and builds artifacts
+
+on:
+  workflow_call:
+    inputs:
+      platform:
+        description: 'Platform name (e.g., ubuntu:noble, rockylinux:8)'
+        type: string
+        required: true
+      architecture:
+        description: 'Architecture (x86_64, aarch64)'
+        type: string
+        required: true
+      sha:
+        type: string
+        required: true
+      redis-ref:
+        type: string
+        required: true
+      beta-version:
+        type: string
+        required: true
+      skip-condition:
+        type: string
+        default: 'false'
+        description: 'Condition to skip this build job'
+
+jobs:
+  # Get configuration for this specific platform and architecture
+  get-config:
```

Comment:
same comment as on `task-test.yml`. Let's avoid adding the `*-with-config` variants